dynawo/sources/Models/Modelica/Dynawo/Electrical/Sources/IEC/WT4Injector.mo

```diff
@@ -73,6 +75,9 @@ model WT4Injector "Converter model and grid interface according to IEC N°61400-
   Modelica.ComplexBlocks.Interfaces.ComplexOutput uWtPu(re(start = u0Pu.re), im(start = u0Pu.im)) "Complex voltage at grid terminal in pu (base UNom)" annotation(
     Placement(visible = true, transformation(origin = {110, -80}, extent = {{-10, -10}, {10, 10}}, rotation = 0), iconTransformation(origin = {110, 60}, extent = {{-10, -10}, {10, 10}}, rotation = 0)));
 
+  Types.ActivePowerPu PGenPu(start = -P0Pu) "Active power at WTT in pu (base SnRef) (generator convention)";
+  Types.ReactivePowerPu QGenPu(start = -Q0Pu) "Reactive power at WTT in pu (base SnRef) (generator convention)";
```

Comment:
WTT ? at terminal ?

Reply:
Yes, WTT means Wind Turbine Terminal. Is "terminal" enough?